android/sign.cmd

```diff
@@ -0,0 +1 @@
+jarsigner -keystore .\eburger-play-key\upload-keystore.jks .\app\build\outputs\bundle\release\app-release.aab eburger-upload-keystore
```

Comment:
this is referencing eburger :)

Reply:
Yes we are using the same Play store account and the same key store / upload key. Back in the day someone named them to be eBurger, but should be just "Sphereon". Renaming...

Reply:
The keystore itself is not in git. The alias in it is still "eburger-upload-keystore" though. There might be a way to rename that, but this seems kind of trivial.